benchmark-service/benchmark/templates/index.html

```diff
@@ -1,11 +1,56 @@
-
 {% extends "base.html" %}
-{% block title %}
-Posts index
-{% endblock %}
+{% block title %} Benchmark {% endblock %}
 
 {% block content %}
 <h1>Benchmark System</h1>
+
+<form action="{{ base_path }}/dabuhijl/benchmark/lookup" method="post">
+    <input type="text" name="file"></input>
+    <input type="submit" value="lookup"></input>
```

Comment:
So the way CSRF protection works as you have a hidden field here that expects there's a context field `csrf_token`.

```
<input type="hidden" name="_csrf" value="{{ csrf_token }}">
```